### PR TITLE
Add more details to Helm rollback window

### LINKED ISSUE #583: Helm rollback window with more details
**What would you like to be added**:
When we use helm rollback via Apps -> Releases it's hard to tell to which revision we would like to roll it back as there is only release number and chart version - would be great to add to the dropdown menu App Version and update date same as the output of "helm history xyz"

**Why is this needed**:
To make Lens more useful in regards to helm

**Environment you are Lens application on:**

- Kubernetes distribution: AWS/Kops
- Desktop OS: Fedora

### PR DESCRIPTION
**Pull request by vshakirova**
Screen Shot 2021-02-04 at 12 22 15 PM

Fixes: https://github.com/lensapp/lens/issues/583

### CODE REVIEW

File: src/renderer/api/endpoints/helm-releases.api.ts [hunk 1]
```diff
@@ -57,6 +57,7 @@ export interface IReleaseRevision {
   updated: string;
   status: string;
   chart: string;
+  app_version: string;
```
Comment (Nokel81) [inline src/renderer/api/endpoints/helm-releases.api.ts:60]: We use camel case for interface field names
```suggestion
  appVersion: string;
```

**Review by Nokel81 — COMMENTED** [review]
Looks good except for this minor change

File: src/renderer/api/endpoints/helm-releases.api.ts (hunk 1 above)
Comment (vshakirova) [reply src/renderer/api/endpoints/helm-releases.api.ts:60]: @Nokel81 I tried `appVersion` but it didn't work. I checked API response and in response they used app_version
Screen Shot 2021-02-04 at 7 45 22 PM
Tell me please if I miss smth

Comment (Nokel81) [reply src/renderer/api/endpoints/helm-releases.api.ts:60]: Oh I didn't realize that this interface was representing the data gotten directly.

**Review by Nokel81 — APPROVED** [review]

**Review by ixrock — APPROVED** [review]

**Pull request merged**